cpp/src/tsne/distances.cuh

```diff
@@ -64,9 +69,10 @@ void get_distances(const raft::handle_t &handle,
                      cudaStream_t userStream,
  */
 
-  raft::spatial::knn::brute_force_knn(handle, input_vec, sizes_vec, input.d,
-                                      input.X, input.n, k_graph.knn_indices,
-                                      k_graph.knn_dists, k_graph.n_neighbors);
+  raft::spatial::knn::brute_force_knn(
+    handle, input_vec, sizes_vec, input.d, input.X, input.n,
+    k_graph.knn_indices, k_graph.knn_dists, k_graph.n_neighbors, true, true,
+    nullptr, raft::distance::DistanceType::L2SqrtExpanded);
```

Comment:
Can we put this metric type in a `DEFAULT_METRIC` constant? It would help us centralize it so it's immediately obvious and collected in one place should we ever need to bother changing it in the future.